src/sections/Community/Members-grid/index.js

```diff
@@ -32,6 +74,7 @@ const MembersGrid = props => {
                   value={props.members}
                   onChange={props.handleChange}
                   theme={dropdownTheme}
+                  styles={colourStyles}
```

Comment:
@Jashpatel1 Even if I pass `colourStyles = {}` to `styles` prop in the react-select component, its behaving that way. Not sure why.

Reply:
Never mind. 😓 fixed it!